docs/skill_json.md

```diff
@@ -6,31 +6,40 @@ This file contains metadata about the skill and is all you need to submit a skil
 
 ```javascript
 {
-  "name": "Wolfram Alpha",
-  "description": "Get information from Wolfram Alpha",
-  "skillname": "skill-wolfie",
-  "authorname": "JarbasSkills",
-  "url": "https://github.com/JarbasSkills/skill-wolfie",
-  "branch": "v0.1",
-  "desktopFile": false,
-  "systemDeps": false,
-  "download_url": "https://github.com/JarbasSkills/skill-wolfie/archive/v0.1.tar.gz",
-  "examples": [
-    "ask the wolf what is the speed of light",
-    "How tall is Mount Everest?",
-    "When was The Rocky Horror Picture Show released?",
-    "What is Madonna's real name?",
-    "What's 18 times 4?",
-    "How many inches in a meter?"
-  ],
-  "lang": "en-us",
-  "lang_support": {
-    "pt-pt": {
-      "name": "Wolfram Alpha",
-      "description": "...",
-      "examples": [...]
+    "name": "Wolfram Alpha",
+    "description": "Get information from Wolfram Alpha",
+    "skillname": "skill-wolfie",
+    "authorname": "JarbasSkills",
+    "url": "https://github.com/JarbasSkills/skill-wolfie",
+    "branch": "v0.1",
+    "desktopFile": false,
+    "systemDeps": false,
+    "download_url": "https://github.com/JarbasSkills/skill-wolfie/archive/v0.1.tar.gz",
+    "examples": [
+        "ask the wolf what is the speed of light",
+        "How tall is Mount Everest?",
+        "When was The Rocky Horror Picture Show released?",
+        "What is Madonna's real name?",
+        "What's 18 times 4?",
+        "How many inches in a meter?"
+    ],
+    "lang": "en-us",
+    "lang_support": {
+        "pt-pt": {
+          "name": "Wolfram Alpha",
+          "description": "...",
+          "examples": [...]
+        }
+    },
+    "requirements": {
+      "qt_version": "5",
```

Comment:
Would it be possible we need other optional qt components? We also have python and other dependencies here, so maybe:

```yaml
requirements:
  python: []
  system: {}
  skill: []
  plugins: [] # Or dict of plugin type to entrypoint?
  qt:
    version: '5'  # Or maybe separate min_version and max_version?
    packages: []  # Or plugins? Not sure the terminology here or if this would ever vary from defaults?
```